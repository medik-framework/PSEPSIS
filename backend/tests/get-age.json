{"id":"tabletApp","tid":1,"interface":"TabletApp","name":"Instruct","args":["get age"]}
{"id":"tabletApp","tid":3,"interface":"TabletApp","name":"Instruct","args":["Enter patient weight"]}
