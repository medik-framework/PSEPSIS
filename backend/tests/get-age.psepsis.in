[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartScreening",
    "eventArgs": [],
    "waitFor":
      { "name" : "StartGUI"
      }
  },
  { "action": "exit" }
]
