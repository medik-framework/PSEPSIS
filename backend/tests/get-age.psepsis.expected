[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  }
]
